genital confliction

TiberianErin opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

adds a new, entertaining pill that contains both types of genital enlargement at overdose levels. this causes them to battle within the user's body over which one grows and which one shrinks. basically a time-consuming genital randomizer.

## Why It's Good For The Game

it's a fun drug and will make my character look like the best chemist ever by turning these homemade pills into a product for a company. (they used to be made by my character in-game. now I'm trying to make them official.)

## A Port?

absolutely not, this is homemade.

## Changelog
:cl:
add: added genital confliction pills / bodily confliction pills
add: added genital confliction pill bottles / bodily confliction pill bottles
tweak: kinkmate sells genital confliction bottles alongside the normal enlargement.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TiberianErin committed:
bodily confliction pills

the other players found it weird when I made it in game.
well, time to sell it to the kinkmate company.

TiberianErin committed:
added genital confliction pill bottle

doesn't work without the pill included

TiberianErin committed:
kinkmate sells body confliction pills

btw body confliction pills are basically just genital growth randomizers.

iknowdude00 commented on code/modules/reagents/reagent_containers/pill.dm:
```diff
@@ -265,3 +265,8 @@
+/obj/item/reagent_containers/pill/genital_conflictions
+	name = "genital confliction pill"
+	desc = "One of the strangest concoctions ever thought up by Euan Morrison, a pill containing both penis enlargement and breast enlargement reagents at a high dosage. This causes an... interesting reaction when administered."
+	list_reagents = list(/datum/reagent/fermi/penis_enlarger = 20, /datum/reagent/fermi/breast_enlarger = 20)
```
You can literally just mix them both and make the pills yourself. Why would we want to add more hornycode stuff when we're aiming to tone that down?

TiberianErin replied:
> You can literally just mix them both and make the pills yourself. Why would we want to add more hornycode stuff when we're aiming to tone that down?

i can and do make the pills myself in-game, i'm just making code for it because it would be interesting lore.
a local chemist's new drug interests the company who's reagents they're mixing together, and pay for it to be added to the vendor's inventory.

Fajetti replied:
> i can and do make the pills myself in-game, i'm just making code for it because it would be interesting lore. a local chemist's new drug interests the company who's reagents they're mixing together, and pay for it to be added to the vendor's inventory.

![unknown (1)](https://user-images.githubusercontent.com/88633279/173411743-a2ee4962-cc32-42d8-a212-1eb19d825681.png)

TiberianErin replied:
h a m p t e r

TiberianErin committed:
Realistic Bicycle Prices

Seriously. Why is it so expensive in the first place? Real Bicycles are somewhere between $400-500.

TiberianErin committed:
Acceptable Bicycle Price

Bicycle price changed to $1000
$10000 to $450 to $1000

TiberianErin committed:
jerking off really fast is possible now

get the horny burst over with so we can get back to work

TiberianErin committed:
added funny descriptions to some of these mobs

yep

TiberianErin committed:
added fart

fart like a goon

TiberianErin committed:
added better fart

let one rip m8

TiberianErin committed:
Merge branch 'Lumos-SS13:master' into master

TiberianErin committed:
more text on it

i put more text inside

TiberianErin committed:
added "desearch"

we forget things in this station

TiberianErin committed:
added additional silly tips

kinda silly

TiberianErin committed:
blibloboblibibli

biblobobabibobibri

TiberianErin committed:
bibobluuuuuhyuuuubabblobibrili

TiberianErin committed:
yoinked goonstation fart noise

for the new '*fart' emote

TiberianErin committed:
audible fart

'u ' smells n i c e

TiberianErin committed:
added god pen admin tool

contains so much fuckin force it'd kill god itself
and it was bought by john wick at your local store

Fajetti commented:
Are you actively being retarded or trying desperately to be funny?


https://user-images.githubusercontent.com/88633279/173477692-7842a005-da27-47ab-ae4d-4dc429649386.mov

TiberianErin commented:
https://user-images.githubusercontent.com/46800751/173487962-aefd2f5e-f890-4ff1-8a8f-4da3b3123caf.mov
